Sources/SwiftSyntax/generated/syntaxNodes/SyntaxDeclNodes.swift

```diff
@@ -15,6 +15,26 @@
 // MARK: - AccessorDeclSyntax
 
 
+
+/// ### Grammar
+/// 
+/// (``AttributeSyntax`` | ``IfConfigDeclSyntax``) `*`
+/// ``DeclModifierSyntax``?
+/// (`'get'` | `'set'` | `'didSet'` | `'willSet'` | `'unsafeAddress'` | `'addressWithOwner'` | `'addressWithNativeOwner'` | `'unsafeMutableAddress'` | `'mutableAddressWithOwner'` | `'mutableAddressWithNativeOwner'` | `'_read'` | `'_modify'` | `'init'`)
+/// ``AccessorParameterSyntax``?
+/// ``AccessorEffectSpecifiersSyntax``?
+/// ``AccessorInitEffectsSyntax``?
+/// ``CodeBlockSyntax``?
```

Comment:
Is there a way to generate the translation between the type/protocol names and the actual grammar terms/components that they represent? I think having such a translation table can help a lot to contributors.

Reply:
What do you mean by “actual grammar terms/components that they represent”?

Reply:
I might be misunderstanding it. What I mean is: it seems to me that not all syntax nodes in the parser correlate one-to-one with the syntax nodes as documented in TSPL.

Reply:
No, there isn’t always a 1:1 correlation between the SwiftSyntax nodes and the grammar rules in TSPL.